src/test/java/com/rakovets/course/java/core/practice/oop/principles/SiameseTest.java

```diff
@@ -0,0 +1,53 @@
+package com.rakovets.course.java.core.practice.oop.principles;
+
+import com.rakovets.course.java.core.practice.oop.principles.Cat;
+import com.rakovets.course.java.core.practice.oop.principles.Siamese;
+import com.rakovets.course.java.core.practice.oop.principles.Sphynx;
+import com.rakovets.course.java.core.practice.oop.principles.Persian;
+import com.rakovets.course.java.core.practice.oop.principles.Person;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class SiameseTest {
+
+    static Stream<Arguments> ProviderArgumentsForSiamese() {
+        return Stream.of(
+                Arguments.of("Varya", "Varya murr-murr-murr meow-meow"),
+                Arguments.of("Tiger", "Tiger murr-murr-murr meow-meow"),
+                Arguments.of("Leon", "Leon murr-murr-murr meow-meow"),
+                Arguments.of("Murka", "Murka murr-murr-murr meow-meow")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("ProviderArgumentsForSiamese")
+    void siameseTest(String str1, String expected) {
+        Siamese siamese = new Siamese(str1);
+        String actual = siamese.getName() + " " + siamese.catPurr() + " " + siamese.catSayMew();
+        Assertions.assertEquals(expected, actual);
```

Comment:
"Violation: There isn't separating empty lines for GIVEN, WHEN, THEN parts
Solution: Add blank line"